src/reporter.js

```diff
@@ -213,6 +218,7 @@ Reporter.prototype = {
         } else if (test.attempts === test.maxAttempts - 1) {
           // Is this our last attempt ever? Then mark the test as finished and failed.
           result.environments[test.profile.id].status = "fail";
+          result.environments[test.profile.id].error = test.stdout;
```

Comment:
can we change `result.environments[test.profile.id].error` to `result.environments[test.profile.id].log`, will it impact admiral?

Reply:
We can change it to "log" or whatever we want. It just goes into mongodb.

Example POST to mongodb via admiral2-api:
```
{ test: 'test/automation/tests/fail.js',
  environments: 
   { chrome: 
      { retries: 0,
        resultURL: 'http://somebuildurl',
        sessionId: 'bfc91c0f9ae0af2430e8bd88446cab09',
        status: 'fail',
        log: 'some error message'
      }
   }
}
```